soapboxpb/environment.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+package soapbox;
+option go_package = "soapboxpb";
+
+import "application.proto";
```

Comment:
This can be deleted, there's no reference to any service or message defined in the application.proto file in this file.